base/applications/osk/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@ file(GLOB osk_rc_deps res/*.*)
 add_rc_deps(rsrc.rc ${osk_rc_deps})
 add_executable(osk main.c settings.c rsrc.rc)
 set_module_type(osk win32gui UNICODE)
-add_importlibs(osk comdlg32 shell32 user32 gdi32 advapi32 comctl32 msvcrt kernel32)
+add_importlibs(osk comdlg32 shell32 user32 gdi32 advapi32 comctl32 msvcrt kernel32 ntdll)
```

Comment:
Unneeded.

Reply:
Uhm, Thomas wanted it, not me (see one of his reviews above).
> Some kind of trace or dprint would be good.

Reply:
Hmmmmmmmmm OK ...
Well we never do these LoadImage checks when setting main application icon in our other GUI apps.